scripts/modules/MsIndustryLinks/templates/ingest/customconnector/flow_customconnector.json

```diff
@@ -0,0 +1,92 @@
+{
+    "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "$connections": {
+        "defaultValue": {},
+        "type": "Object"
+      },
+      "$authentication": {
+        "defaultValue": {},
+        "type": "SecureObject"
+      }
+    },
+    "triggers": {
+      "manual": {
+        "type": "Request",
+        "kind": "Button",
+        "inputs": {
+          "schema": {
+            "type": "object",
+            "properties": {
+              "payload": {
+                "title": "Payload",
+                "type": "string",
+                "x-ms-dynamically-added": true,
+                "description": "The array of items to ingest into Dataverse.",
+                "x-ms-content-hint": "TEXT"
+              }
+            },
+            "required": ["payload"]
+          }
+        }
+      }
+    },
+    "actions": {
+      "Map_data": {
```

Comment:
Is the output of this used anywhere?

Reply:
Yes, the output of Map_data has to be configured in the parameters for custom connectors that accept an array as input. This cant be directly configured in the workflow because the input name of the custom connector can be variable, so hardcoding that into the workflow isn't ideal.